packages/shared.test/src/handlers/loadObjectsEndpoints.ts

```diff
@@ -653,10 +692,12 @@ async function handleStreamPoints(
       | BaseAPIError
     >
   >[0],
+  useGeotime?: boolean,
 ) {
   const requestBody = await req.request.json();
-  const streamPointsResp =
-    streamPointsRequestHandlers[stableStringify(requestBody)];
+  const streamPointsResp = useGeotime
```

Comment:
The new API is designed to support normal timeseries as well, so once I add that support, we won't need to bifurcate and can just call the same API without needing to worry about backend implementation